enterprise/internal/completions/streaming/dotcom/dotcom.go

```diff
@@ -0,0 +1,90 @@
+package dotcom
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/sourcegraph/sourcegraph/enterprise/internal/completions/types"
+	"github.com/sourcegraph/sourcegraph/internal/httpcli"
+	streamhttp "github.com/sourcegraph/sourcegraph/internal/search/streaming/http"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+var PROVIDER = "dotcom"
```

Comment:
```suggestion
var ProviderName = "dotcom"
```

all upper case seems uncommon in go, but maybe that's just me